src/style.css

```diff
@@ -2,0 +1,68 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
You should provide a fallback font for 'Avenir' in case it fails to load. This can be a system font like 'Arial', 'Helvetica', or 'sans-serif'. (CSS checklist)